README.md

```diff
@@ -1 +1,45 @@
-# awesome-code-connector
\ No newline at end of file
+<div align="center">
+<a href="https://codeconnector.io/" target="_blank">
+	<img src="assets/ccLogos/codeconnectorLogo.png" alt="codeconnector">
+</a>
+	<br>
+	<h4>Open Source Learning Resources, compiled by the Dev Community for the Dev Community.</h4>
```

Comment:
Why is this an h4 without an h1-3 on the page?

Reply:
was for sizing, will change to an h1.